tests/unit/type/list.tcl

```diff
@@ -1136,8 +1136,6 @@ foreach {pop} {BLPOP BLMPOP_LEFT} {
             {rpush k hello}
             {pexpireat k *}
             {swapdb 1 9}
-            {select 9}
-            {del k}
```

Comment:
to make things more explicit, let's add some commands after swapdb and watch their propagation here.
i.e. we can do `set somekey someval` and then `get k` so we can see lazy expiry happens after swapdb.
let's do the same in the test below.

and let's mention this change (notouch+noexpire) in the top comment (to make it clear why the tests had to be changed)

Reply:
ohh, i see your next commit relies on`set-active-expire`. i'd rather my approach (one block of assert_replication_stream)
also, the one with active expire seems like it has a race condition.

Reply:
@oranagra check my last commit (new one) Is that what you meant?

Reply:
@ranshid i suggested to set another key before the EXISTS, so that we can distinguish that expiration from a case in which it was expired by SWAPDB.

Reply:
@oranagra please check [78c0d4d](https://github.com/redis/redis/pull/11012/commits/78c0d4d97c0a7ee9772ba0f11939c9096f5866c4)